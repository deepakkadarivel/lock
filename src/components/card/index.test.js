import React from 'react';
import { create } from 'react-test-renderer';
import Card from './index';

describe('Card component', () => {
    test('Renders Card component', () => {
        const component = create(
            <Card
                text="card test content"
                subText="card sub test content"
                icon="add"
                onClick={() => {}}
                onDelete={() => {}}
            />
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
