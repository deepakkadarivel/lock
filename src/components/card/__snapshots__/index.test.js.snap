// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component Renders Card component 1`] = `
<div
  className="Card"
  onClick={[Function]}
  role="presentation"
>
  <span
    className="Card-data"
  >
    <i
      className="material-icons Card-icon"
    >
      add
    </i>
    <div>
      <p
        className="Card-data--text"
      >
        card test content
      </p>
      <p
        className="Card-data--subText"
      >
        card sub test content
      </p>
    </div>
  </span>
  <span
    className="Card-action"
  >
    <i
      className="material-icons t-delete"
      onClick={[Function]}
      role="presentation"
    >
      delete
    </i>
  </span>
</div>
`;
